config/bidderinfo.go

```diff
@@ -276,8 +276,11 @@ func processBidderAliases(aliasNillableFieldsByBidder map[string]aliasNillableFi
 		if err := validateAliases(aliasBidderInfo, bidderInfos, bidderName); err != nil {
 			return nil, err
 		}
+
 		//required for CoreBidderNames function to also return aliasBiddernames
-		openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName)))
+		if err := openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName)), openrtb_ext.BidderName(aliasBidderInfo.AliasOf)); err != nil {
```

Comment:
`bidderName` need be typecasted to string as it is already a string. We have done similar changes in [this commit](https://github.com/prebid/prebid-server/commit/ebb3adc92258095e69433abb4c5df10dcd6b7725)

Reply:
This should be done in the `merge master` commit.